src/g2ccsv.c

```diff
@@ -177,6 +177,32 @@ g2c_find_entry(char *desc, G2C_CODE_TABLE_T *table)
     return NULL;
 }
 
+/** Implementation of strsep for code portability.
+ * Extracts first token in string given a delimiter.
+ *
+ * @param stringp The address of a pointer to string to be separated.
+ * This value is overwritten with the value past the delimiter.
+ * @param delim Characters that delimit the tokens.
+ *
+ * @author Alyson Stahl @date 11/6/24
+ *
+ * @return a pointer to the original value of stringp
+ */
+char *
```

Comment:
Can you make this function static?

Then it will only be visible to this file.